src/Parser/Paladin/Protection/Modules/Features/Abilities.js

```diff
@@ -39,7 +39,7 @@ class Abilities extends CoreAbilities {
     {
       spell: SPELLS.SHIELD_OF_THE_RIGHTEOUS,
       category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
-      getCooldown: haste => null,
+      getCooldown: haste => 16,
```

Comment:
I'm pretty sure SotR scales with haste. Also if there is anything else that modifies cooldown, it should go here.

Reply:
your right. it does scale with haste. Will make changes.